Sources/PackageGraph/PackageGraph.swift

```diff
@@ -33,8 +33,12 @@ public struct PackageGraph {
         return rootPackages.flatMap{$0.modules}.contains(module)
     }
 
+    /// The errors encountered while loading the package graph.
+    public let errors: [Swift.Error]
```

Comment:
Looks good for now, but eventually we will also want to handle warnings, notices (e.g. about upgrading or fixing issues), etc.  In short, a more general notion of diagnostics.